app/_src/policies/meshtimeout.md

```diff
@@ -71,6 +73,9 @@ MeshTimeout policy lets you configure multiple timeouts:
 - http streamIdleTimeout
 - http maxStreamDuration
 - http maxConnectionDuration
+{% if_version gte:2.6.x %}
+- http requestHeadersTimeout
+{% endif_version %}
 
 ### Timeouts explained
```

Comment:
This change breaks a little the view of `Timeouts explained` header as it's indented too much in `2.5.x` and earlier versions

![image](https://github.com/kumahq/kuma-website/assets/11655498/4922729c-56fc-4630-bc72-b3d416cb4a9d)

Reply:
Might be because of the dangling ```{% endif_version %}``` I pointed out